pnpm-lock.yaml

```diff
@@ -28115,7 +28479,6 @@ packages:
   /escape-string-regexp@2.0.0:
     resolution: {integrity: sha512-UpzcLCXolUWcNu5HtVMHYdXJjArjsF9C0aNnquZYY4uW/Vu0miy5YoWvbV345HauVvcAUnpRuhMMcqTcGOY2+w==}
     engines: {node: '>=8'}
-    dev: true
```

Comment:
I don't know what to make of these `dev: true` lines disappearing, given we're only changing dev-dependencies in the PR. At this point I suspect some pnpm bug? Still, the fact that they're just disappearing and not changing to `dev: false` makes me think it's probably ok...? Something like "this is not only a dev dependency (`dev: true`) nor only a prod dependency (`dev: false`), but transitive through both kinds"? Which still doesn't explain how this PR could cause that kind of change, but oh well.

Reply:
Oh, I see, the change at least makes more sense now. I suspect that should be a dev-dep, I'll look into it.